NAMESPACE

```diff
@@ -9,8 +11,10 @@ import(MultiAssayExperiment)
 importFrom(AnnotationHub,query)
 importFrom(ExperimentHub,ExperimentHub)
 importFrom(ExperimentHub,loadResources)
+importFrom(MultiAssayExperiment,experiments)
 importFrom(S4Vectors,DataFrame)
 importFrom(SingleCellExperiment,SingleCellExperiment)
 importFrom(SpatialExperiment,SpatialExperiment)
+importFrom(SummarizedExperiment,SummarizedExperiment)
```

Comment:
@drighelli 
Please add this to the DESCRIPTION file. 'SummarizedExperiment' needs to be in the 'Imports' field. Thank you.